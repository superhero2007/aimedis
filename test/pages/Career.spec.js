import React from 'react';
import { shallow } from 'enzyme';

import Career from 'pages/Career';

function setup() {
  return shallow(<Career />);
}

describe('Career', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
