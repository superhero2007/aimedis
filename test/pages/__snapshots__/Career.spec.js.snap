// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Career should render properly 1`] = `"<div class=\\"app__career app__route\\"><div class=\\"container\\"><section class=\\"app__career__section1\\"><div class=\\"app__career__section1__content\\"><div class=\\"row\\"><div class=\\"col-12\\"><div class=\\"app__career__section1__content__info\\"><p class=\\"app__career__section1__content__info__title\\">Careers</p><p class=\\"app__career__section1__content__info__subtitle\\">Open positions</p></div></div></div></div></section><section class=\\"app__career__section2\\"><div class=\\"app__career__section2__content\\"><div class=\\"row\\"><div class=\\"col-12 app__career__section2__content__item\\"><div class=\\"app__career__item\\"><div class=\\"app__career__item__content\\"><div class=\\"app__career__item__content__info\\"><p class=\\"app__career__item__content__info__title\\">Web devoloper</p><p class=\\"app__career__item__content__info__subtitle\\">React native / JS, Node JS, NoSQL, PHP</p><div class=\\"app__career__item__content__info__icon\\"><span><i class=\\"i-map-marker\\"></i> VENLO</span><span><i class=\\"i-clock-o\\"></i> FULL-TIME</span></div></div><div class=\\"app__career__item__content__button\\"><a href=\\"/career/1\\" class=\\"btn btn__primary\\">READ MORE</a></div></div></div></div><div class=\\"col-12 app__career__section2__content__item\\"><div class=\\"app__career__item\\"><div class=\\"app__career__item__content\\"><div class=\\"app__career__item__content__info\\"><p class=\\"app__career__item__content__info__title\\">Web devoloper</p><p class=\\"app__career__item__content__info__subtitle\\">React native / JS, Node JS, NoSQL, PHP</p><div class=\\"app__career__item__content__info__icon\\"><span><i class=\\"i-map-marker\\"></i> VENLO</span><span><i class=\\"i-clock-o\\"></i> FULL-TIME</span></div></div><div class=\\"app__career__item__content__button\\"><a href=\\"/career/2\\" class=\\"btn btn__primary\\">READ MORE</a></div></div></div></div><div class=\\"col-12 app__career__section2__content__item\\"><div class=\\"app__career__item\\"><div class=\\"app__career__item__content\\"><div class=\\"app__career__item__content__info\\"><p class=\\"app__career__item__content__info__title\\">Web devoloper</p><p class=\\"app__career__item__content__info__subtitle\\">React native / JS, Node JS, NoSQL, PHP</p><div class=\\"app__career__item__content__info__icon\\"><span><i class=\\"i-map-marker\\"></i> VENLO</span><span><i class=\\"i-clock-o\\"></i> FULL-TIME</span></div></div><div class=\\"app__career__item__content__button\\"><a href=\\"/career/3\\" class=\\"btn btn__primary\\">READ MORE</a></div></div></div></div><div class=\\"col-12 app__career__section2__content__item\\"><div class=\\"app__career__item\\"><div class=\\"app__career__item__content\\"><div class=\\"app__career__item__content__info\\"><p class=\\"app__career__item__content__info__title\\">Web devoloper</p><p class=\\"app__career__item__content__info__subtitle\\">React native / JS, Node JS, NoSQL, PHP</p><div class=\\"app__career__item__content__info__icon\\"><span><i class=\\"i-map-marker\\"></i> VENLO</span><span><i class=\\"i-clock-o\\"></i> FULL-TIME</span></div></div><div class=\\"app__career__item__content__button\\"><a href=\\"/career/4\\" class=\\"btn btn__primary\\">READ MORE</a></div></div></div></div><div class=\\"col-12 app__career__section2__content__item\\"><div class=\\"app__career__item\\"><div class=\\"app__career__item__content\\"><div class=\\"app__career__item__content__info\\"><p class=\\"app__career__item__content__info__title\\">Web devoloper</p><p class=\\"app__career__item__content__info__subtitle\\">React native / JS, Node JS, NoSQL, PHP</p><div class=\\"app__career__item__content__info__icon\\"><span><i class=\\"i-map-marker\\"></i> VENLO</span><span><i class=\\"i-clock-o\\"></i> FULL-TIME</span></div></div><div class=\\"app__career__item__content__button\\"><a href=\\"/career/5\\" class=\\"btn btn__primary\\">READ MORE</a></div></div></div></div></div></div></section><section class=\\"app__career__section3\\"><div class=\\"app__career__section3__content\\"><div class=\\"row app__career__section3__content__culture\\"><div class=\\"col-12 col-lg-6 app__career__section3__content__info\\"><div><p class=\\"app__career__section3__content__info__title\\">Our Culture</p><p class=\\"app__career__section3__content__info__description\\">Soon Aimedis will offer much more including AIMSocial - the interactive patient platform that is secured and supported by a blockchain and help.</p></div></div><div class=\\"col-12 col-lg-6\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div><div class=\\"row app__career__section3__content__mission\\"><div class=\\"col-12 col-lg-6\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"col-12 col-lg-6 app__career__section3__content__info\\"><div><p class=\\"app__career__section3__content__info__title\\">Our Mission</p><p class=\\"app__career__section3__content__info__description\\">Soon Aimedis will offer much more including AIMSocial - the interactive patient platform that is secured and supported by a blockchain and help.</p></div></div></div></div></section><section class=\\"app__career__mask d-none d-lg-block\\"><div class=\\"app__career__mask__one\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"app__career__mask__two\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></section></div></div>"`;
